### PR TITLE
Feature zipcode button and lat long stub

### PR DESCRIPTION
**Pull request by c-h-russell-walker**
Curious to hear your feedback on this @pbouzakis :)

### CODE REVIEW

File: app/js/ui/CreateAccount.js [hunk 1]
```diff
@@ -51,4 +61,56 @@ export default class CreateAccount extends React.Component {
...
+                        console.log(lat);
+                        console.log(lng);
+                    } catch (e) {
+                        console.error(e);
+                    }
+                }
+            } else {
+                // window.alert('Geocoder failed due to: ' + status);
+            }
+        });
```
Comment (pbouzakis) [inline app/js/ui/CreateAccount.js]: This is something we should move to an action to keep the view/controller layer simple.

**Comment by pbouzakis** [comment]
Looks good.

In a future PR I'd like to move this to an action. I dont like to have much logic in the view controllers.

**Comment by pbouzakis** [comment]
Let's get this in master, and then we can refactor out into actions.

**Comment by c-h-russell-walker** [comment]
Sounds good to me :)

Currently setting up a google sheet for our "backlog" - so as we iterate on these features we can put ideas for improvements or changes to them in the backlog:
https://docs.google.com/spreadsheets/d/1aIIX2xhgtJKArVnxS7dHj05PJz-kNdCmefF-zhqOnvk/edit#gid=0

**Comment by pbouzakis** [comment]
Merged into master.

**Pull request closed without merging**